hail_search/queries/snv_indel.py

```diff
@@ -73,6 +74,13 @@ class SnvIndelHailTableQuery(MitoHailTableQuery):
         PATHOGENICTY_HGMD_SORT_KEY: lambda r: [MitoHailTableQuery.CLINVAR_SORT(CLINVAR_KEY, r), r.hgmd.class_id],
     }
 
+    FREQUENCY_PREFILTER_FIELDS = OrderedDict([
+        (True, PREFILTER_FREQ_CUTOFF),
```

Comment:
why is af of 1% handled differently than 3, 5, and 10%? why don't we store it in our reference data?

Reply:
this isn't reading the reference data, its reading the `high_af_variants` prefilter table. This is a way of quickly filtering out high frequency variants without reading in the whole table by instead anti-joining with a table that has all the varianst that are too frequent to return in search. Everything in the prefilter table has a frequency of at least 0.1%, so that threshhol is treated differently because you can just filter on the existance of the row in the table and not a specific annotation on the row